src/commands/containers.rs

```diff
@@ -0,0 +1,370 @@
+use std::path::Path;
```

Comment:
can this not be inside bin?

Reply:
I believe we'd have to turn off autobin discovery, and then manually register our binaries. I'm fine with that, if you are:
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#target-auto-discovery

Reply:
No it seems to work to just move the commands mod you placed in `src/commands` in `src/bin/commands` instead and fix the imports so that they are public, could probably make all cross mods public

Reply:
Ah ok, I thought you meant move them to the root directory. Will do shortly, I've also done some refactoring inside docker.